.pre-commit-config.yaml

```diff
@@ -1,7 +1,22 @@
-files: 'shapely/'
+ci:
+  autoupdate_schedule: quarterly
+exclude: ^versioneer\.py$
 repos:
+  - repo: https://github.com/pre-commit/pre-commit-hooks
+    rev: v4.6.0
+    hooks:
+    - id: check-yaml
+    - id: end-of-file-fixer
+    - id: trailing-whitespace
   - repo: https://github.com/astral-sh/ruff-pre-commit
-    rev: "v0.5.2"
+    rev: v0.5.2
     hooks:
+      # Run the linter.
+      - id: ruff
+        args: [ --fix ]
+      # Run the formatter.
       - id: ruff-format
```

Comment:
And thanks for fixing this! I was planning to open a PR as well, as the comments about non-sorted imports without actually fixing them were becoming annoying ;)